Mathlib/Algebra/GroupWithZero/NonZeroDivisors.lean

```diff
@@ -277,3 +275,78 @@ lemma nonZeroSMulDivisors_mulOpposite_eq_op_nonZeroDivisors :
     Rᵐᵒᵖ ⁰[R] = R⁰.op := rfl
 
 end nonZeroSMulDivisors
+
+noncomputable section Associated
+
+open nonZeroDivisors
+
+/-- The units of the monoid of non zero divisors of `α` are equivalent to the units of `α`. -/
+def nonZeroDivisorsUnitsEquiv (α : Type*) [MonoidWithZero α] :
+    (α⁰)ˣ ≃* αˣ :=
```

Comment:
This has nothing to do with `Associates`, right?

Reply:
Indeed, but that's not a problem. right?

Reply:
All I'm saying is that the section name is inappropriate if it only refers to half the declarations within it 😄

Reply:
Yes, that's true. I'll change it